.php-cs-fixer.dist.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-$config = PhpCsFixer\Config::create();
+$config = new PhpCsFixer\Config();
 $config->getFinder()
     ->exclude('vendor')
     ->in(__DIR__);
```

Comment:
```suggestion
    ->in(__FILE__)
    ->in(__DIR__);
```

Reply:
If I do this, then I get:
`The "/home/phil/git/sabre-io/cache/.php-cs-fixer.dist.php" directory does not exist.`

`in()` takes a directory/directories.

Reply:
In sabre-io/xml we put:
```
    ->append([
        __FILE__,
    ]);
```

I guess that makes cs-fixer parse-and-fix-format of its own `.php-cs-fixer.dist.php` code.